legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/depot/api/DepotMetadataApi.java

```diff
@@ -46,7 +46,7 @@
     private final JsonMapper jsonMapper = JsonMapper.builder().addMixIn(Entity.class, EntityMixIn.class).build();
 
     private static final String GET_ENTITIES_PATH = "/api/projects/%s/%s/versions/%s";
-    private static final String GET_DEPENDENCIES_PATH = "/api/projects/%s/%s/versions/%s/projectDependencies";
+    private static final String GET_DEPENDENCIES_PATH = "/api/projects/%s/%s/version/%s/projectDependencies";
```

Comment:
This looks like it's maybe a typo in the Depot APIs. Maybe we should see about fixing it there instead of changing it here.

Reply:
I changed that back and added a fix to legend-depot